File: adapter/src/main/java/com/nathaniel/adapter/adapter/OnItemChildClickListener.java
package com.nathaniel.adapter.adapter;

import android.view.View;

/**
 * @author Nathaniel
 * @version V1.0.0
 * @datetime 2020/4/19 - 16:58
 */
public interface OnItemChildClickListener {
    boolean onItemChildClick(BaseRecyclerAdapter<?> adapter, View view, int position);
}

File: adapter/src/main/java/com/nathaniel/adapter/adapter/OnItemChildClickListener.kt
package com.nathaniel.adapter.adapter

import android.view.View

/**
 * @author Nathaniel
 * @version V1.0.0
 * @package com.example.adapterapplication.adapter
 * @datetime 2020/4/19 - 16:58
 */
interface OnItemChildClickListener {
    fun onItemChildClick(adapter: BaseRecyclerAdapter<*>?, view: View?, position: Int): Boolean
}